devices/wholeBodyDynamics/WholeBodyDynamicsDevice.cpp

```diff
@@ -1029,6 +1239,60 @@ bool WholeBodyDynamicsDevice::attachAllFTs(const PolyDriverList& p)
         return false;
     }
 
+    // Attach the controlBoardList to the controlBoardRemapper
+    bool ok = remappedMASInterfaces.multwrap->attachAll(ftSensorList);
+    ok = ok & remappedMASInterfaces.multwrap->attachAll(tempSensorList);
+
+    if( !ok  )
+    {
+        yError() << " WholeBodyDynamicsDevice::attachAll in attachAll of the remappedMASInterfaces";
+        return false;
+    }
+
+    // Check if the MASremapper and the estimator have a consistent number of ft sensors
+    int tempSensors = 0;
+    tempSensors=static_cast<int>( remappedMASInterfaces.temperatureSensors->getNrOfTemperatureSensors());
+    if( tempSensors > static_cast<int>( estimator.sensors().getNrOfSensors(iDynTree::SIX_AXIS_FORCE_TORQUE)) )
+    {
+        yError() << "wholeBodyDynamics : The multipleAnalogRemappedDevice has more sensors than those in the open estimator ft sensors list";
+        return false;
+    }
+
+    // Temporary code to check name of sensors, assuming the temperature sensor name will be the same as the ft sensor
+    int checkCounter=0;
+    std::string ftName;
+    std::string tempName;
+    int ftMap=-1;
+    //for (auto & sensorName : tempDeviceNames){
```

Comment:
Could you remove this stranded line?

Reply:
I was trying to prepare for moving to newer C++ standard but yeah it doesn't make sense to be there for nothing. Deleting